src/components/common/Button.tsx

```diff
@@ -141,20 +134,18 @@ const Button = ({
   fullWidth,
   disabled,
   loading,
-  href,
-  onClick,
+  ...buttonAttributes
 }: ButtonProps) => {
-  return (
-    <ButtonElement
-      href={href}
-      onClick={onClick}
-      className={button({
-        variant,
-        modifier,
-        size,
-        fullWidth,
-        disabled,
-      })}>
+  const classes = button({
```

Comment:
To have less repetition down below I store it in a variable here. `buttonClasses` would've probably been a better name I realise now.